lib/rules/forbid-foreign-prop-types.js

```diff
@@ -109,7 +109,7 @@ module.exports = {
             && !ast.isAssignmentLHS(node)
             && !isAllowedAssignment(node)
           )) || (
-            // @ts-expect-error The JSXText type is not present in the estree type definitions
+            // @ts-expect-error: Literal is not a valid type
```

Comment:
O.o why is Literal not a valid type?

Reply:
Sorry. It's `JSXText` that's not a valid type.

```
(node.property.type === 'Literal' || node.property.type === 'JSXText')
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                     ^: This comparison appears to be unintentional because the 
                                        types '"ArrayExpression" | "ArrowFunctionExpression" | 
                                        "AssignmentExpression" | "AwaitExpression" | 
                                        "BinaryExpression" | "CallExpression" | "ChainExpression" | 
                                        "ClassExpression" | ... 16 more ... | "PrivateIdentifier"' 
                                        and '"JSXText"' have no overlap. 
                                        ts(2367)
```

Reply:
I have reverted this comment change.